portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsParagraphField.testcase

```diff
@@ -51,12 +51,50 @@ definition {
 	}
 
 	@description = "Verify that a Paragraph Field can be duplicated"
-	@ignore = "true"
 	@priority = "5"
 	test CanBeDuplicated {
+		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-131224 CanBeDuplicated pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Paragraph");
+
+		FormFields.editParagraphBody(content = "Test");
+
+		Click(
+			key_name = "Paragraph",
+			locator1 = "FormFields#FIELD_TARGET_LABEL");
+
+		Click(locator1 = "Icon#VERTICAL_ELLIPSIS_BUTTON");
+
+		Click(
+			key_selectOption = "Duplicate",
+			locator1 = "FormViewBuilder#SELECT_FIELD_OPTIONS_LIST");
+
+		Form.save();
+
+		FormsAdminNavigator.gotoBack();
+
+		FormsAdminNavigator.gotoEditFormByName(formName = "Form 1");
+
+		AssertElementPresent(
+			key_bodyText = "Test",
+			key_fieldName = "Paragraph",
+			locator1 = "FormFields#FIELD_TARGET_LABEL_AND_BODY_TEXT");
+
+		AssertElementPresent(
+			key_bodyText = "Test",
+			key_fieldName = "Copy of Paragraph",
+			locator1 = "FormFields#FIELD_TARGET_LABEL_AND_BODY_TEXT");
 	}
```

Comment:
Hi @LucasAcacio, 

I am not sure but I think this last step on the test is missing --> 
10. Assert the new Paragraph field's field reference is different from the field reference of the original Paragraph field. 

Can you verify this, please? 

Thanks!